incubator/nodejs-loopback/templates/scaffold/package-lock.json

```diff
@@ -0,0 +1,4327 @@
+{
+  "name": "nodejs-loopback-scaffold",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
```

Comment:
Is it a good idea to include a package-lock file in the template?

Personally, I think developers using appsody should be getting the most recent version of all dependencies that are available at install (scaffold) time. To achieve that with a package-lock file, we would have to frequently update it (e.g. by setting up a https://renovatebot.com).

Can we simply remove the package-lock file from the template, at least for now?

Reply:
I agree with this comment. Even though there is a likelihood that latest version might be the one stack provider has tested on, it is preferred that they get the latest versions within the rules allowed by package.json.